flink-runtime/src/test/java/org/apache/flink/runtime/execution/librarycache/BlobLibraryCacheManagerTest.java

```diff
@@ -504,6 +506,41 @@ public void closingLibraryCacheManager_closesClassLoader() throws IOException {
 		assertTrue(classLoader.isClosed());
 	}
 
+	@Test
+	public void releaseUserCodeClassLoader_willRunReleaseHooks() throws Exception {
```

Comment:
```suggestion
	public void releaseUserCodeClassLoader_willRunReleaseHooks() throws IOException, InterruptedException {
```
maybe nit but it will safe a bit from IDE warnings which might be useful in other cases

Reply:
I see, ok